main.cc

```diff
@@ -762,6 +762,15 @@ int main(int ac, char** av) {
                 mscfg.encrypt = netw::messaging_service::encrypt_what::rack;
             }
 
+            if (clauth && (mscfg.encrypt == netw::messaging_service::encrypt_what::dc || mscfg.encrypt == netw::messaging_service::encrypt_what::dc)) {
+                startlog.warn("Setting require_client_auth is incompatible with 'rack' and 'dc' internode_encryption values."
+                    " It is possible for an internode connection to pretend to be in the same rack/dc by spoofing"
+                    " its broadcast address in the handshake and bypass authentication. To ensure that mutual TLS"
+                    " authentication is not bypassed, please set internode_encryption to 'all'. Continuing with"
+                    " insecure configuration."
+                );
```

Comment:
This is too strongly worded. In many environments, such spoofing is impossible (for example, public clouds).

Reply:
Well, in origin (where the text is from), remote address is not determined on socket level, but by what the connection claims to be from (protocol). So spoofing is very much possible there. But since we're doing connection level, I agree it can be less strong here.

Reply:
Changed the text...